py/desispec/pipeline/run.py

```diff
@@ -95,12 +95,26 @@
 ]
 
 
-def qa_path(datafile, suffix="_QA"):
+def qa_path(datafile, suffix="_QA", figformat='pdf'):
+    '''
```

Comment:
I suggest you add a qaformat keyword as well.  Not sure
we'll stick with YAML for the rest of time.  
Even consider hooking into desispec.io.qa 
to get the format from there..